resources/views/dashboard.blade.php

```diff
@@ -26,8 +26,8 @@
             </section>
         </div>
 
-        <div id="app" class="bg-frost font-sans relative z-0">
-            <div class="max-w-4xl mx-auto py-6">
+        <div id="app" class="bg-frost font-sans relative overflow-x-scroll z-0">
```

Comment:
This _probably_ isn't necessary to set overflow anymore if we change the word break behaviors, but `auto` is probably better since it won't show the empty scroll bar at the bottom if scrolling isn't necessary.

```suggestion
        <div id="app" class="bg-frost font-sans relative overflow-x-auto z-0">
```